tools/render-symbols.js

```diff
@@ -57,7 +58,7 @@ This is used internally to generate historic version data for Chrome's APIs.
   /** @type {chromeTypes.ProcessedAPIData} */
   const o = JSON.parse(await getStdin());
 
-  const fq = new FeatureQueryAll(o.feature);
+  const fq = new FeatureQuery(o.feature);
```

Comment:
```suggestion
  const featureQuery = new FeatureQuery(o.feature);
```

JS is not Go ;-)

Reply:
Just realized that this was already there, but still might be good to rename.

Reply:
Hah, no worries. Renamed.